### ISSUE TITLE
Deff SME Needs modification

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#7600]
(WEB REPORT BY: tifora REMOTE: 158.69.55.2:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 691b1769f6aa95756ea24d3f02656894a167333a

Deff's SME set up is designed to fail and needs to be changed. The gas filters need to be either removed or altered since the moment the canisters reach 4500kpa the entire cooling loop shuts down. The coolant out is dangerous as hell since hot oxygen and plasma is pumped into a tiny section of pipe which upon reaching 500 degrees auto ignites and clogs the loop with co2.

**Comment by ghost** [comment]
@Duny-

**Comment by ihadtoregisterforthis** [comment]
Of course the engine will need maintenance or upgrading, that's why there's an **engine**ering department.

**Comment by Duny-** [comment]
The meh basic setup I had in place was perfectly functional before faggots complained and nexis changed it.
If you want the engine changed again show me the setup you would like to see because frankly I don't know what you niggers want. It has to be under the average setup an engineer could make in efficiency and just enough to charge the SMES cells for the round ideally (while leaving room for modifications), not something ideal that doesn't need improvements and is easily started up. The whole point of moving away from the singularity/easily achievable huge AME was for engineers to have more to do before they fuck off to explore space.
Keep in mind you don't HAVE TO keep a thousand emitters always on the shard, the original SME setup on bay required to fire on the supermatter a few times then turn it off. My former setup worked in a similar way.
You also don't have to keep the engine running after having charged all SMES cells to full, but I know that requires coming back to engineering and is therefore impossible so let's forget that.

tl;dr: I have no problem with changing the current setup but you need to show me a setup that is much less than ideal yet functional enough to power the station for a round after setting it up and allows enough room for knowledgeable engineers to improve it a lot.

**Comment by Probe1** [comment]
> Deffs setup is designed to fail

![1416762590023](https://cloud.githubusercontent.com/assets/8550416/14248938/4647fd98-fa45-11e5-9abe-17c3bbf74c1e.gif)

I'll work on it.

**Comment by DamianX** [comment]
Is this still a thing?

**Comment by PJB3005** [comment]
Probably not and probably never has been.

**Comment by Probe1** [comment]
I even wrote a guide on how to set up the SME on Deff by removing 2 pipes and adding 5.  

If you have trouble on Deff then wire the solars.
